lib/routes/yilin/index.ts

```diff
@@ -0,0 +1,58 @@
+import { Data, DataItem, Route } from '@/types';
```

Comment:
Please use [namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) of `yilinzazhi` instead of `yilin`